src/content/old-cleopatra-profile-format.js

```diff
@@ -329,37 +330,6 @@ export function convertOldCleopatraProfile(
   return {
     meta: Object.assign({}, meta, { version: CURRENT_VERSION }),
     threads: threads.map(t => _convertThread(t, symbolicationTable)),
-    tasktracer: {
-      taskTable: {
-        length: 0,
-        dispatchTime: [],
-        sourceEventId: [],
-        sourceEventType: [],
-        parentTaskId: [],
-        beginTime: [],
-        processId: [],
-        threadIndex: [],
-        endTime: [],
-        ipdlMsg: [],
-        label: [],
-        address: [],
-      },
-      tasksIdToTaskIndexMap: new Map(),
-      stringArray: [],
-      addressTable: {
-        length: 0,
-        address: [],
-        className: [],
-        lib: [],
-      },
-      addressIndicesByLib: new Map(),
-      threadTable: {
-        length: 0,
-        tid: [],
-        name: [],
-        start: [],
-      },
-      tidToThreadIndexMap: new Map(),
-    },
+    tasktracer: emptyTaskTracerData(),
   };
```

Comment:
I mean this function's return value is typed `Object` but we should return `Profile` instead. I tried but there are new flow errors coming after this so I thought it's better to do it in a separate patch.